py3-cffi.yaml

```diff
@@ -0,0 +1,44 @@
+# Generated from https://pypi.org/project/cffi/
+package:
+  name: py3-cffi
+  version: 1.15.1
+  epoch: 0
+  description: Foreign Function Interface for Python calling C code.
+  copyright:
+    - license: MIT
+  dependencies:
+    runtime:
+      - py3-pycparser
+      - python-3.11
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - wolfi-base
+      - busybox
+      - build-base
+      - python-3.11
+      - python-3.11-dev
+      - py3.11-setuptools
+      - libffi-dev
+
+pipeline:
+  - uses: fetch
+    with:
+      README: 'CONFIRM WITH: curl -L https://files.pythonhosted.org/packages/2b/a8/050ab4f0c3d4c1b8aaa805f70e26e84d0e27004907c5b8ecc1d31815f92a/cffi-1.15.1.tar.gz | sha256sum'
```

Comment:
`README` isn't a valid option for the [fetch](https://github.com/chainguard-dev/melange/blob/main/pkg/build/pipelines/fetch.yaml) pipeline.  I'm guessing this comes from the `melange convert` tool when creating python related packages?  We don't validate the schema which would help automatically help catch things like this TBH.

I think it's worth dropping the `README:` option for all these packages as they will become out of date after the next wolfi auto update.